### ISSUE TITLE
Not compatible with protractor 3.0 

### ISSUE THREAD

**Issue by nivithaCisco** [jintoppy/protractor-html-screenshot-reporter#74]
The base folder gets deleted after execution . This is happening after upgrading to protractor 3.0

**Comment by laixiangran** [comment]
+1

**Comment by sukeerthm** [comment]
Same issue, New version of protractor is not supporting "protractor-html-screenshot-reporter". for time being i switched to "protractor-jasmine2-screenshot-reporter".

**Comment by tyleragnew** [comment]
Same issue.

**Comment by jsolmosi** [comment]
same problem. I use protractor-jasmine2-screenshot-reporter instead but there isn't provided any info about total passed/failed tests and environment(like browser/os) in report. Is it possible to set it up?

**Comment by sukeerthm** [comment]
I guess you are using older version(0.1.7), please install latest version (0.2.0). Its capturing test execution summary and config details.

**Comment by jsolmosi** [comment]
yes you are right, my bad. I confused protractor-jasmine2-screenshot-reporter with protractor-jasmine2-html-reporter

**Comment by satyasrikanth** [comment]
It would be very helpful if this is fixed

**Comment by iljapavlovs** [comment]
+1

**Comment by Dilip9922** [comment]
+1

**Comment by louiealmeda** [comment]
+1

**Comment by kummerer94** [comment]
+1

**Comment by Nikhil0538** [comment]
+1

**Comment by Istenes** [comment]
+1

**Comment by cristinaRaluca** [comment]
+1

**Comment by Divyata-Singh** [comment]
I get the following error " Please pass a valid base directory to store the screenshots into". The options parameter of ScrenshotReporter constructor (in index.js) remains undefined irrespective of whatever values I pass from the conf file.

**Comment by adelitofarias** [comment]
+1
Does not work with my version 3.3.0 of the protractor in my Ubuntu 14.04

**Comment by kryz81** [comment]
+1

**Comment by mkalpana** [comment]
+1

**Comment by HomeboxJim** [comment]
omg I thought **I** was going crazy!

**Comment by TheAksh** [comment]
+1

**Comment by celiawchan** [comment]
+1

**Comment by anujchandran** [comment]
+1 not working in protractor 4.0.9

**Comment by edgarechm** [comment]
I have Protractor 4.0.10 and node 6.9.1, also NOT compatible.

When running script, I get an exception error here:
npm\node_modules\protractor\node_modules\q\q.js:155      ----> throw e;
